src/types/input.ts

```diff
@@ -9,6 +9,6 @@ export type InputBaseProps<TValue> = {
     suffix?: string;
     status?: Validation;
     valueSelect?: boolean;
-    onChange?: (value: TValue) => void;
+    onChange?: (value?: TValue) => void;
```

Comment:
If you pass 0 you can revert this.

Reply:
But the issue with `0` is what happens when there input is cleared?  Being a `number` input `0` is a valid value.  We must be able to pass a `undefined`.